spec/dummy/app/controllers/admin/series_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Admin::SeriesController < Alchemy::Admin::ResourcesController
+
+end
```

Comment:
Could you please add a new line at the end of the file and remove the empty line inside the class body? I updated the Rubocop settings to also include the dummy app, so would have been catched by HoundCI. Thanks